source/_docs/energy/individual-devices.markdown

```diff
@@ -22,3 +22,19 @@ Smart relays sit behind your "normal" switches and make them smart. It allows yo
 Some smart devices, such as air conditioning, boilers, and others, may provide a power sensor, measured in Watts. You can use the [Integration (Riemann sum integral) integration](/integrations/integration/#energy) to calculate the energy your device is using. You can then use the energy sensor in the Energy Dashboard, as individual devices. For information on setting up an entity for use in the **Energy** dashboard, refer to the [energy FAQ](/docs/energy/faq/#troubleshooting-missing-entities).
 
 <img src='/images/docs/energy/devices.png' alt='Graphic showing energy flowing from the home to individual devices.' style='border: 0;box-shadow: none; display: block; max-height: 400px; margin: 0 auto;'>
+
+## Upstream devices and hierarchies
+
+You can create a hierarchy of devices by setting one device as an "upstream device" of another. This means you can now create a device hierarchy within your energy configuration, establishing parent-child relationships between devices.
```

Comment:
```suggestion
You can create a hierarchy of devices by setting one device as an "upstream device" of another. This means you can now establish parent-child relationships between devices within your energy configuration.
```

To avoid having "you can create a device hierarchy" repeated